assemble/bin/accumulo-cluster

```diff
@@ -632,22 +467,16 @@ function main() {
   accumulo_cmd="${bin}/accumulo"
   SSH='ssh -qnf -o ConnectTimeout=2'
 
-  # Copy input arguments into new array
-  # removing any options
-  DEBUG=0
-  i=0
-  declare -a program_args
-  for arg in "$@"; do
-    if [[ $arg == "--dry-run" ]]; then
-      DEBUG=1
-    else
-      program_args[i++]="$arg"
-    fi
-  done
+  cmd="$1"
+  shift
+  parse_args "$@"
 
-  debug "debug: ${DEBUG} args: ${program_args[*]}"
+  localhost_names=("localhost" "127.0.0.1" "$(hostname)" "$(hostname -a 2>/dev/null | head -1)" "$(hostname -s)" "$(hostname -f)")
+  read -r -a localhost_interfaces <<<"$(hostname -I)"
+  LOCAL_HOST_ADDRESSES+=("${localhost_names[@]}" "${localhost_interfaces[@]}")
+  debug "${LOCAL_HOST_ADDRESSES[*]}"
```

Comment:
We could make this conditional on some variable not existing, allowing the user to supply their own localhost addresses.

Reply:
I like that approach, It would be helpful for situations where a CNAME is being used instead of something directly tied to the local host.

Once we decide on what variable to use I could backport that specific override functionality to 2.1 in #5094

Reply:
@ddanielr - I added `ACCUMULO_LOCALHOST_ADDRESSES` in 822f6bf. This allows the user to specify the localhost addresses for the current host. This may only be useful when doing start-here or stop-here.

Reply:
@ctubbsii made a suggestion to just remove the localhost handling and always `ssh` to the host, even if it's the localhost. That would simplify the script to be sure, with the small performance penalty of using `ssh` locally. It would require that password-less `ssh` be set up correctly on the host, but I think that's already a prerequisite for Hadoop. Thoughts on that?

Reply:
> @ctubbsii made a suggestion to just remove the localhost handling and always `ssh` to the host, even if it's the localhost. That would simplify the script to be sure, with the small performance penalty of using `ssh` locally. It would require that password-less `ssh` be set up correctly on the host, but I think that's already a prerequisite for Hadoop. Thoughts on that?

I think that's fine for the `start`& `stop` behavior cluster wide behavior if that's already a requirement for hadoop.

 Would that be used in conjunction with `ACCUMULO_LOCALHOST_ADDRESSES` var for determining the `stop-here` `start-here` target?

Reply:
I'm thinking actually that `ACCUMULO_LOCALHOST_ADDRESSES` may not work. I think what we really want is a call to `nslookup`, `getent` or similar. Removing the localhost processing totally I think will make this work as it will try to `ssh` to the address in cluster.yaml regardless of whether it's the local host or not, and address resolution will take place in the execution of `ssh`.